scripts/cfgCommons.py

```diff
@@ -40,8 +40,7 @@ def __init__(self, args):
         commonSection = 'common'
 
         # List of attributes of framework section to be appended rather then ovewrriden (if the same parameter is defined in more config files)
-        frameworkAppend = ['xls', 'intents', 'entities', 'dialogs', 'functions', 'generated_intents',
-                           'generated_entities', 'generated_dialog']
+        frameworkAppend = ['xls', 'intents', 'entities', 'dialogs', 'functions']
```

Comment:
generated_* fields are not an array by their nature. We need to know one specific place to put generated files into.

Reply:
My intuition is, that naturally folders intents, entities and dialogs are arrays, because we want it to be easy to include more reused content. It might seem natural to allow multiple generated dialog,... folders too. However, their purpose is to tell what "temporary" folders do we use as a destination for T2C output. There is no point (I am aware of) to have multiple output destinations for T2C outputs. One is enough to pass the data to later stages of the build.

Reply:
I  could imagine both: 
- inputs -arrays / outputs- strings only;  Entities, intents, dialogs are  arrays as they are "inputs only " to scripts. There can be multiple of them.  generated_* are (also) outputs - therefore  we need a single path to clearly say where they go. If we generate intents/entities/dialogs from different sources, we would invent new word for it
 - keep the same representation of all path; in case of generic_* we wold use only first element of it. 

As second option seems to work for some time, I suggest to keep it unless we find another argument to switch

Reply:
on @Tereza-Pytelova confirmation question: I keep my opinion :-) , both are fine, let us not change what is not broken.

Reply:
For now leaving as it is, we can discuss further in #237